src/SettingsModal.js

```diff
@@ -109,6 +114,22 @@ const SettingsForm = ({setTimers, ...props}) => {
             <span className="text-danger">{errors.longBreak}</span>
           </Row>
         </Form.Group>
+        <Form.Group>
+          <Row>
+            <Form.Label>Example select</Form.Label>
+            <Form.Control
+              as="select"
+              value={get(settings, 'sound', '/ring.mp3')}
+              onChange={onTimeChangeFn('sound')}
+            >
+              <option value={'/ring.mp3'}>Sound 1</option>
+              <option value={'/notification_bell.mp3'}>Sound 2</option>
+              <option value='/notification.mp3'>Sound 3</option>
+              <option value='/notification_sound.mp3'>Sound 4</option>
+              <option value='/notification_sound_1.mp3'>Sound 5</option>
```

Comment:
Give names to these sounds like:

"Fon fon" or "ring ring ring"

something meaningful as labels